drivers/tty/serial/8250/8250_port.c

```diff
@@ -459,25 +458,6 @@ static void io_serial_out(struct uart_port *p, int offset, int value)
 	outb(value, p->iobase + offset);
 }
 
-#ifdef CONFIG_SERIAL_8250_TS
```

Comment:
Is this CONFIG_ option still otherwise used? If not, that should also be removed from Kconfig files.